src/components/styles/Button.styled.ts

```diff
@@ -6,7 +6,7 @@ export const BaseButton = styled.div<{ fontSize?: string; isActive?: boolean }>`
   align-items: center;
   .ant-btn-text {
     color: ${(props) => props.theme.palette.common.white};
-    font-size: ${(props) => props.fontSize};
+    font-size: ${(props) => `clamp(14px, 2vw, ${props.fontSize})`};
```

Comment:
might want to rename max font size?

Reply:
or use themes for this

Reply:
6f83ddcef630951d836d70309f969b0953101e6e